### PR TITLE
stricter `Booleanish` type (`react`, `babel-plugin-react-html-attrs`, and `rax`)

### PR DESCRIPTION
**Pull request by brunocrosier**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/rax/index.d.ts [hunk 1]
```diff
@@ -25,16 +25,7 @@ type NativePointerEvent = PointerEvent;
 type NativeTransitionEvent = TransitionEvent;
 type NativeUIEvent = UIEvent;
 type NativeWheelEvent = WheelEvent;
-type Booleanish = boolean | 'true' | 'false';
-
-/**
```
Comment (brunocrosier) [inline types/rax/index.d.ts:30]: not used or exported

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## babel-plugin-react-html-attrs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/babel-plugin-react-html-attrs@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'babel-plugin-react-html-attrs'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
## react ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react@latest/))
was missing the following properties:
1. unstable_act
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against e11dda64d1cf6609054b396fed29081207979e52
</p>

**Review by eps1lon — CHANGES_REQUESTED** [review]
Could you elaborate why this is needed? template literal types are a newish TypeScript feature so this might needlessly break users with older TypeScript versions.

**Comment by brunocrosier** [comment]
> Could you elaborate why this is needed? template literal types are a newish TypeScript feature so this might needlessly break users with older TypeScript versions.

Sure! In my opinion ```type Booleanish = `${boolean}` ``` is preferable to `type Booleanish = "true" | "false"` because it is:

1) shorter
2) less prone to future typos (e.g. it will never be assignable to `"true" | "flase"`)

Template literal types are used widely elsewhere throughout this repo. I would argue that this is a good example of why they exist.

**Pull request closed without merging**